src/Microsoft.Azure.SignalR/ServerConnections/ClientConnectionContext.cs

```diff
@@ -22,6 +22,16 @@
 
 namespace Microsoft.Azure.SignalR
 {
+    /// <summary>
+    /// Client Application Layer     Transport      Application
+    /// ========================     =========      ===========
+    /// |                      |     |        |     |         |
+    /// |      |-------|-------|-----|-Input--|-----|--Output |
+    /// |                      |     |        |     |         |
+    /// |      |-------|-------|-----|-Output-|-----|--Input  |
+    /// |                      |     |        |     |         |      
+    /// ========================     =========      ===========
+    /// </summary>
```

Comment:
A good comment, I like it :)

Reply:
Add `->` to show the direction of data flow?

Reply:
`<` has special meaning in triple-slash comments 😢

Reply:
move out of the ///?

Reply:
use emoji arrow....